Orange/data/dask.py

```diff
@@ -150,6 +178,12 @@ def compute(self) -> Table:
         table.name = self.name
         return table
 
+    def sample(self, size: int = DEFAULT_SAMPLE_SIZE):
+        # TODO: how is the random seed handled in Orange elsewhere?
```

Comment:
Usually we want to have workflows reproducible, so set a random seed (like you already did).

I usually instantiate it is a `np.random.RandomState` and did not even know about the `default_rng` function.